src/pages/experimental/empty-state-pattern/index.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Empty state
+---
+
+<PageDescription>
+
+_Empty states_ have several uses. A common use is where something is yet to be populated e.g. a table. Another common use is when a search or filter yields no results.
+
+</PageDescription>
+
+#### Status:
+
+[Experimental](/experimental/overview/)
+
+#### Maintainer:
+
+[Vikki Paterson](https://github.com/vikkipaterson), [Chloe Poulter](https://github.com/chloepoulterdesign)
+
+![Contextual image](images/0.png)
+
+## Empty state consists of
+
+- An illustration that is relevant to the context, the product or generic
+- Headline text that explains why the state is empty
```

Comment:
```suggestion
- A headline that explains what information is missing and why
```

Reply:
I'm not sure about this one - when used in a search, there isn't information missing, so much as information that has been searched/filtered out.
@vikkipaterson what do you think about this wording?

Reply:
I have made a suggestion because I think the point is not that information is missing it is explaining why the state is empty.